types/integrate-adaptive-simpson/vector.d.ts

```diff
@@ -0,0 +1,2 @@
+import integrate = require('.');
```

Comment:
What does this file do?

Reply:
@jessetrinity the `/vector` file exports a similar method, but instead returns an array of numbers. This has now been updated to reflect the correct type